Dockerfile

```diff
@@ -24,9 +25,15 @@ RUN \
     && wget -qO - https://privatebin.info/key/release.asc | gpg2 --import - \
     && rm -rf /var/www/* \
     && cd /tmp \
-    && wget -qO ${RELEASE}.tar.gz.asc ${PBURL}releases/download/${RELEASE}/PrivateBin-${RELEASE}.tar.gz.asc \
-    && wget -q ${PBURL}archive/${RELEASE}.tar.gz \
-    && gpg2 --verify ${RELEASE}.tar.gz.asc \
+    && if expr "${RELEASE}" : '[0-9]\{1,\}\.[0-9]\{1,\}\.[0-9]\{1,\}$' >/dev/null ; then \
+         echo "getting release ${RELEASE}"; \
+         wget -qO ${RELEASE}.tar.gz.asc ${PBURL}releases/download/${RELEASE}/PrivateBin-${RELEASE}.tar.gz.asc \
+         && wget -q ${PBURL}archive/${RELEASE}.tar.gz \
+         && gpg2 --verify ${RELEASE}.tar.gz.asc ; \
+       else \
+         echo "getting tarball for ${RELEASE}"; \
+         wget -qO ${RELEASE}.tar.gz ${PBURL}tarball/${RELEASE}; \
```

Comment:
Am I wrong or would this lead to the buildx script requesting https://github.com/PrivateBin/PrivateBin/tarball/nightly ? Which does not work…

Reply:
In the nightly build, The RELEASE is set the the git reference (ie master). If the RELEASE is a semantic version, it will download the release.

`nightly` is the tag for the docker image  name.

Reply:
You were right in that the buildx.sh script used the wrong order.